report.go

```diff
@@ -58,6 +71,11 @@ func writeError(w io.Writer, err error, p reportParams) {
 		var v string
 		if u, ok := value.(Unquoted); ok {
 			v = string(u)
+		} else if s, ok := value.(SuppressedIfLong); ok {
+			v = p.format(s.Value)
+			if !testingVerbose() && strings.Count(v, "\n") >= longValueLines {
```

Comment:
Rather than calling through the global variable (something we've been quite careful to avoid in other tests, cf `WithVerbosity`), could we not maybe add the verbosity to `reportParams` somehow?